frontend/awx/common/SurveyStep.tsx

```diff
@@ -53,16 +53,14 @@ export function SurveyStep({
   );
 
   useEffect(() => {
-    const survey: { [key: string]: string | number | { name: string }[] } = {};
+    const survey: { [key: string]: string | string[] | number | { name: string }[] } = {};
```

Comment:
Do we still need to include `{ name: string }[]` in the survey key:value type?

Reply:
probably not, I'll verify